benchmarks/ASQA/README.md

```diff
@@ -1,55 +1,141 @@
 # ASQA BENCHMARK
 
-## Description
+## 1. Description
 
 This benchmark is designed to evaluate the performance of the [ASQA dataset](https://huggingface.co/datasets/din0s/asqa). For `generate.py`, we followed [FLARE](https://github.com/jzbjyb/FLARE), using `gpt-3.5-turbo-instruct` with no retrieval settings.
```

Comment:
To generate RAG results, we followed [FLARE](https://github.com/jzbjyb/FLARE), using `gpt-3.5-turbo-instruct` with no retrieval settings, as implemented in `generate.py`(+ link).